tests/utils/getScrollTop.test.js

```diff
@@ -0,0 +1,15 @@
+import { getScrollTop } from '../../index'
+
+describe('getScrollTop', () => {
+    it('returns the scroll offset distance to 10', () => {
+        expect(getScrollTop()).toBe(Math.abs((window.scrollY || window.scrollTop || document.getElementsByTagName('html')[0].scrollTop, 10)))
```

Comment:
This code does not really test anything, as it simply repeats the code from `getScrollTop` as the expected value.  I think what we should test here is if `scrollY` and `scrollTop` have certain values, does the function return the correct absolute value? So in the test, we should mock `scrollY` and `scrollTop` and test if `getScrollTop` returns the correct value.